node/core/backing/src/lib.rs

```diff
@@ -121,7 +121,7 @@ impl ValidatedCandidateCommand {
 }
 
 /// The candidate backing subsystem.
-pub struct CandidateBackingSubsystem<RuntimeInfo: RuntimeInfoProvider> {
+pub struct CandidateBackingSubsystem<RuntimeInfo> {
```

Comment:
nit: I would have kept the bound here on the struct definition, IMO. It makes it more clear what the requirements are when seeing this struct for the first time.

Reply:
Thanks for the link. Looks like that's mostly about derivable traits. For other traits it says this:

> There is a grey area around other non-derivable trait bounds that are not strictly required by the structure definition, like Read or Write. They may communicate the intended behavior of the type better in its definition but also limits future extensibility. Including semantically useful trait bounds on data structures is still less problematic than including derivable traits as bounds.

I don't really see why it would be problematic in this case.